src/components/layouts/Settings.tsx

```diff
@@ -163,24 +165,19 @@ export function Settings(props: Props) {
       disableSaveButton={props.disableSaveButton}
     >
       <div className="grid grid-cols-12 lg:gap-10">
-        <div className="col-span-12 md:col-span-4 lg:col-span-3">
+        <div className="col-span-12 lg:col-span-3">
           <a className="flex items-center py-4 px-3 text-xs uppercase font-medium text-gray-600">
             <span className="truncate">{t('basic_settings')}</span>
           </a>
 
-          <SelectField className="lg:hidden">
+          <SelectField
+            className="lg:hidden"
+            value={location.pathname}
+            onValueChange={(value) => navigate(value)}
+            withBlank
+          >
             {basic.map((item) => (
-              <option
-                key={item.name}
-                value={item.href}
-                className={classNames(
-                  item.current
-                    ? 'bg-gray-200 text-gray-900'
-                    : 'text-gray-600 hover:bg-gray-50 hover:text-gray-900',
```

Comment:
Those props have no effect on the option HTML element. That's the reason for the removal.